assets/hmbkp.js

```diff
@@ -125,7 +124,7 @@ function hmbkpToggleScheduleFields( recurrence ) {
 
     var settingFields = jQuery( '.recurring-setting' );
     var scheduleSettingFields = jQuery( '#schedule-start' );
-    var twiceDailyNote = jQuery( 'p.twice-js' );
+    var twiceDailyNote = jQuery( '.twice-js' );
```

Comment:
This change should actually have been part of https://github.com/humanmade/backupwordpress/commit/0abea487e40f10ae4e3155c28ce9b0a7c2c4ca7, I'll leave in here as I assume this is going to be merged before the next release anyway.